modules/osdk-cli-reference-test.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * operators/osdk-cli-reference.adoc
+
+[id='osdk-cli-reference-test-{context}']
+= test
+
+The `operator-sdk test` command has subcommands that can test the Operator
+locally or from within a cluster.
+
+== local
+
+The `local` subcommand runs Go tests built using the Operator SDK's test
+framework locally.
+
+.`test local` arguments
+[options="header",cols="1,3"]
+|===
+|Arguments |Description
+
+|`<test_location>` (string)
+|Location of e2e test files (e.g., `./test/e2e/`).
+|===
+
+.`test local` flags
```

Comment:
The sdk had a new release last week that added 3 new flags to `test local` (`--up-local`, `--no-setup`, and `--image`): https://github.com/operator-framework/operator-sdk/blob/master/doc/sdk-cli-reference.md#local

Reply:
Thx, added!